README.md

```diff
@@ -100,8 +99,11 @@ No modules.
 |------|-------------|------|---------|:--------:|
 | aws\_region | The AWS region to deploy into (e.g. us-east-1). | `string` | `"us-east-1"` | no |
 | findings\_data\_s3\_bucket | The name of the S3 bucket where findings data is stored.  Note that in production workspaces, '-production' is automatically appended this bucket name.  In non-production workspaces, '-<workspace\_name>' is automatically appended to this bucket name. | `string` | n/a | yes |
+| findings\_data\_s3\_users\_group\_name | The base name of the group to be created for findings data S3 bucket access users.  Note that in production workspaces, '-production' is automatically appended this bucket name.  In non-production workspaces, '-<workspace\_name>' is automatically appended to this bucket name. | `string` | `"findings_data_s3_users"` | no |
+| findingsbucketfullaccess\_policy\_description | The description to associate with the IAM policy that allows full access to the findings data S3 bucket. | `string` | `"Allows full access to the S3 bucket where findings data is stored."` | no |
+| findingsbucketfullaccess\_policy\_name | The base name to associate with the IAM policy that allows full access to the findings data S3 bucket.  Note that in production workspaces, '-production' is automatically appended this bucket name.  In non-production workspaces, '-<workspace\_name>' is automatically appended to this bucket name. | `string` | `"FindingsBucketFullAccess"` | no |
 | tags | Tags to apply to all AWS resources created. | `map(string)` | `{}` | no |
-| usernames | The usernames associated with the accounts to be created.  The format first.last is recommended. | `list(string)` | n/a | yes |
+| users | A list of the usernames for the users that should be given access to the S3 bucket that manages Assessment findings data. | `list(string)` | n/a | yes |
```

Comment:
Is it worth mentioning that these users _must_ have already been created via [cisagov/cool-users-non-admin](https://github.com/cisagov/cool-users-non-admin)?

Reply:
This is a good point, but I would rather bake it into the pre-requisites in the README. Thoughts?

Reply:
I'm fine as long as it's mentioned somewhere.

Reply:
The COOL -iam configurations only mention it as a pre-requisite in the README (which is what I'm using as a "standard").

Reply:
Please see [733c715](https://github.com/cisagov/findings-data-s3-users/pull/21/commits/733c715116291cc9aea398bada4b3fff2845537a).